code/datums/diseases/kingstons.dm

```diff
@@ -0,0 +1,46 @@
+/datum/disease/kingstons
+	name = "Kingstons"
+	max_stages = 4
+	spread_text = "Airborne"
+	cure_text = "Milk."
+	cures = list("milk")
+	cure_chance = 75
+	agent = "NYA Virus"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	permeability_mod = 0.75
+	desc = "If left untreated the subject will turn into a felonid. In felines it has..OTHER..effects."
```

Comment:
`felonid` -> `feline`

`In felines it has..OTHER..effects.` -> `In felines it has... OTHER... effects.`

I don't like it, but if you're going to ellipses, use two of them, and you need spaces...